lib/csv.rb

```diff
@@ -952,6 +954,7 @@ def initialize(data,
       row_separator: row_sep,
       quote_character: quote_char,
       quote_empty: quote_empty,
+      write_converters: write_converters,
```

Comment:
Could you use `converters`? `write_` is redundant here. Because this is `@writer_options`.